packages/applications-service-api/src/routes/index.js

```diff
@@ -11,6 +11,7 @@ const dbConnectionTestRouter = require('./db-connection-test');
 const documentRouter = require('./documents');
 const representationsRouter = require('./representations');
 const timetablesRouter = require('./timetables');
+const submissionRouter = require('./submission');
```

Comment:
Not important but should this be submissions in line with the endpoint name.

Reply:
Yes good spot, when creating these new files I had `submission` in my head but you are right it is `/submissions`. Will update